lib/daru/io/importers/importer.rb

```diff
@@ -0,0 +1,38 @@
+module Daru
+  module IO
+    module Importers
+      class Importer
+        def self.guess_parse(keys, vals)
```

Comment:
Pretty strange place for this method. I'd say: either instance method of `Importers::Base`, or some `Util` **module** with methods of generic use.